weave-js/src/components/Panel2/PanelDir/Component.tsx

```diff
@@ -140,14 +142,16 @@ interface SubfileRowProps {
   file: FileMetadata;
   fileName: string;
   path?: string[];
+  config: any;
   setFilePath(path: string[]): void;
 }
 
 const SubfileRow: React.FC<SubfileRowProps> = props => {
-  const {file, fileName, path, setFilePath} = props;
+  const {file, fileName, path, config, setFilePath} = props;
   const newPath = path?.concat([fileName]) ?? [fileName];
   const iconName = iconFromFileName(fileName);
   // const iconName = fileInfo.iconName;
+  const isDownloadable = config?.isDownloadable;
```

Comment:
whats the default for this value? is this false? asking since it looks like on the bottom we use that to show the download icon